### PR TITLE
Pia 3199 no result updates

### PR DESCRIPTION
**Pull request by chriskryn**
Update display of buttons within no result screen

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift [hunk 1]
```diff
@@ -88,13 +88,21 @@ final public class NoResultScreenViewController: UIViewController {
...
     private (set) var dataSource: HelpDataSource
     private var giniConfiguration: GiniConfiguration
     private let tableRowHeight: CGFloat = 44
     private let sectionHeight: CGFloat = 70
     private let type: NoResultType
     private let viewModel: NoResultScreenViewModel
+    private var buttonsHeightConstraint: NSLayoutConstraint?
+    private var numberOfButtons: Int {
+        return [
+            viewModel.isEnterManuallyHidden() == false,
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift:100]: [Suggestion] viewModel.isEnterManuallyHidden() is already a Boolean, it should not be compared to false.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift [hunk 2]
```diff
@@ -88,13 +88,21 @@ final public class NoResultScreenViewController: UIViewController {
...
+            viewModel.isRetakePressedHidden() == false
+        ].filter({
+            $0 == true
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift:103]: Same

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift [hunk 3]
```diff
@@ -128,18 +136,11 @@ final public class NoResultScreenViewController: UIViewController {
...
-        }
-        if viewModel.isRetakePressedHidden() == false {
-            numberOfButtons += 1
-        }
         if numberOfButtons > 0 {
             tableView.contentInset = UIEdgeInsets(
                 top: 0,
                 left: 0,
-                bottom: buttonsView.bounds.size.height + numberOfButtons * GiniMargins.margin,
+                bottom: buttonsView.bounds.size.height + CGFloat(numberOfButtons) * GiniMargins.margin,
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift:143]: [Suggestion] I think the `numberOfButtons` can be declared as CGFloat from the start

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift [hunk 4]
```diff
@@ -205,6 +206,14 @@ final public class NoResultScreenViewController: UIViewController {
         }
     }
 
+    private func getButtonsMinHeight(numberOfButtons: Int) -> CGFloat {
+        if numberOfButtons == 1 {
+            return 50
```
Comment (vizaknaidavid) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift:211]: You should use the constants declared below

**Review by vizaknaidavid — CHANGES_REQUESTED** [review]

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift [hunk 5]
```diff
@@ -205,6 +206,14 @@ final public class NoResultScreenViewController: UIViewController {
         }
     }
 
+    private func getButtonsMinHeight(numberOfButtons: Int) -> CGFloat {
+        if numberOfButtons == 1 {
+            return Constants.singleButtonHeight.rawValue
+        } else {
+            return Constants.twoBUttonsHeight.rawValue
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift:213]: [Suggestion] a little typo twoBUttonsHeight -> twoButtonsHeight 😉

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift [hunk 6]
```diff
@@ -327,7 +314,7 @@ final public class NoResultScreenViewController: UIViewController {
 
     private func configureButtons() {
         configureButtonsColors()
-        addBlurEffect(button: enterButton, cornerRadius: 14)
+        enterButton.addBlurEffect(cornerRadius: 14)
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift:317]: [Suggestion] it makes sense to reuse _outlineButtonCornerRadius_ from _GiniConfiguration_.

**Review by zladzeyka — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/3911816/200593693-13965d85-3731-4a0a-a05d-f82d751ad1e6.MP4
@chriskryn would you mind to check this issue, looks like the old images are retained 🥲

**Review by zladzeyka — APPROVED** [review]
@chriskryn Great job!😊
I'll need your input on the documentation.

**Review by vizaknaidavid — APPROVED** [review]
Looks good

**Review by zladzeyka — APPROVED** [review]
Looks good!

**Pull request merged**